frontend/src/tests/ForestClientInput.spec.ts

```diff
@@ -102,4 +121,286 @@ describe('ForestClientInput', () => {
         // the forest client input should now be empty
         expect(inputField.value).toBe('');
     });
+
+    it('should raise error for forest client number not found', async () => {
+        const input = wrapper.find('#forestClientInput');
+        const inputField: HTMLInputElement = input.element as HTMLInputElement;
+        await input.setValue(TEST_INVALID_FOREST_CLIENT_NUMBER);
+        expect(inputField.value).toBe(TEST_INVALID_FOREST_CLIENT_NUMBER);
+
+        await wrapper.find("[aria-label='Add Client Numbers']").trigger('click');
+        await flushPromises();
+
+        expect(wrapper.find('#forestClientInputValidationError').element.textContent).toBeTruthy();
```

Comment:
let's verify the text content as well, add the validation errors to the constant file, like the part "is inactive and cannot be added."

Reply:
and no card is displayed in all the error cases

Reply:
The reason why I didn't verify the text content for the validation errors in the first place is that it is not recommended in the documentation because the text might change in the future, which would lead to test fail.

If you feel that the validation error texting would be stable enough, we could change this test.

Maybe an alternative would be to check just if the text contains "inactive" or "invalid" words, which I think would not change. What do you think? @MCatherine1994 

Reference: https://vee-validate.logaretm.com/v4/guide/testing/
<img width="864" alt="Screenshot 2023-12-22 at 07 51 30" src="https://github.com/bcgov/nr-forests-access-management/assets/117364634/4ba5a39e-42cc-4038-8510-27509827ccf5">